Sprint-1/destructuring/exercise-1/exercise.js

```diff
@@ -6,7 +6,8 @@ const personOne = {
 
 // Update the parameter to this function to make it work.
 // Don't change anything else.
-function introduceYourself(___________________________) {
+
+function introduceYourself({name,age,favouriteFood}){
```

Comment:
Please, apply proper formatting to your code: add spaces between your variables. It makes your code easier to read and it becomes important once you will be reading a lot of lines of code in your career.

Reply:
Thanks, I updated the code